Library/Homebrew/cmd/audit.rb

```diff
@@ -423,7 +426,7 @@ def audit_homepage
   end
 
   def audit_github_repository
-    return unless @strict
+    return unless @online
```

Comment:
I think this would be better as `return unless @online && @strict`. The reason is these two flags have different semantical meaning.